src/platform/linux/osfiles.cpp

```diff
@@ -50,74 +50,74 @@
 
 #include <iostream>
 
-std::string dll_search_path;
+std::string librarySearchPath;
 
 // NAN_METHOD is a macro; it's shorthand so you don't have to write that the
 // first and only parameter is of type Nan::FunctionCallbackInfo<v8::Value>
-NAN_METHOD(OSFilesSetDllSearchPath)
+NAN_METHOD(OSFilesSetLibrarySearchPath)
 {
     // Parse parameter from the FunctionCallbackInfo received, convert into a std::string
     if (info.Length() > 0) {
         if (info[0]->IsString()) {
             v8::String::Utf8Value param1(info[0]->ToString());
             std::string path = std::string(*param1);
 
-            printf("\nlinux/osfiles.cpp: SetDllSearchPath() called with %s\n\n",
+            printf("\nlinux/osfiles.cpp: setLibrarySearchPath() called with %s\n\n",
                 path.c_str()
             );
 
-            dll_search_path.assign(path);
+            library_search_path.assign(path);
         }
     }
     /// TODO: Add some error throwing if no parameters or the parameter is not a string
 }
 
 
-/* Try to locate a dynamically-linked library named fileName, and set dll_path
+/* Try to locate a dynamically-linked library named fileName, and set libraryPath
  * to the full path to that library.
  */
-errorcode_t OSFilesFindDll(std::string &dll_path, std::string &fileName)
+errorcode_t OSFilesFindLibrary(std::string &libraryPath, std::string &fileName)
 {
-    char temp_dll_path[COMMON_MAX_PATH];
-    memset(temp_dll_path, 0, COMMON_MAX_PATH);
+    char temp_libraryPath[COMMON_MAX_PATH];
```

Comment:
Wait, this is mixed case. Why are we not using either CamelCase (`tempLibraryPath`) or snake_case (`temp_library_path`)?

Reply:
That is the unintended effect of a search replace, should be replaced with tempLibraryPath everywhere. Will fix.